angular/src/components/settings/vault-timeout-input.component.ts

```diff
@@ -19,6 +19,8 @@ export class VaultTimeoutInputComponent implements ControlValueAccessor, Validat
   }
 
   static CUSTOM_VALUE = -100;
+  static MIN_CUSTOM_MINUTES = 1;
```

Comment:
Should the value be 0 on some clients or do we default to not handling `00:00` in the custom timeout option?

Reply:
On web the expectation is to not allow 00:00. The minimum timeout should be a minute.